tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -214,7 +214,7 @@ You can use **physical mode** or **logical mode** to migrate **existing data**.
 Physical mode has the following limitations:
 
 - When you use physical mode, you cannot create a second migration job or import task for the TiDB cluster before the existing data migration is completed.
-- Physical mode is only available for TiDB clusters deployed on AWS.
+- Physical mode is available for TiDB clusters deployed on AWS and Google Cloud.
```

Comment:
This is not considered a limitation if logical mode is also available on AWS and Google Cloud.